docs/setup_guides/cloud.rst

```diff
@@ -147,192 +117,61 @@ This script sets up the Kubernetes `Role Based Access Control
-        limit: 4
-        guarantee: 1
-      memory:
-        limit: 14G
-        guarantee: 4G
-
-    hub:
-      extraConfig:
-        customPodHook: |
-          from kubernetes import client
-          def modify_pod_hook(spawner, pod):
-              pod.spec.containers[0].security_context = client.V1SecurityContext(
-                  privileged=True,
-                  capabilities=client.V1Capabilities(
-                      add=['SYS_ADMIN']
-                  )
-              )
-              return pod
-          c.KubeSpawner.modify_pod_hook = modify_pod_hook
-          c.JupyterHub.logo_file = '/usr/local/share/jupyter/hub/static/custom/images/logo.png'
-          c.JupyterHub.template_paths = ['/usr/local/share/jupyter/hub/custom_templates/',
-                                        '/usr/local/share/jupyter/hub/templates/']
-      image:
-        name: jupyterhub/k8s-hub
-        tag: v0.6
-      extraVolumes:
-        - name: custom-templates
-          gitRepo:
-            repository: "https://github.com/pangeo-data/pangeo-custom-jupyterhub-templates.git"
-            revision: "b09721bb1a1248dc115730d3c8a791600eae257e"
-      extraVolumeMounts:
-        - mountPath: /usr/local/share/jupyter/hub/custom_templates
-          name: custom-templates
-          subPath: "pangeo-custom-jupyterhub-templates/templates"
-        - mountPath: /usr/local/share/jupyter/hub/static/custom
-          name: custom-templates
-          subPath: "pangeo-custom-jupyterhub-templates/assets"
-
-    cull:
-      enabled: true
-      users: false
-      timeout: 1200
-      every: 600
-
-    # this section specifies the IP address for pangeo.pydata.org
-    # remove or change for a custom cluster
-    proxy:
-      service:
-        loadBalancerIP: 35.224.8.169
-
-The other file is ``secret_config.yaml``, which specifies cluster specific
-encryption tokens. The jupyterhub proxy secret token is just a random hash, 
-which you can generate as follows.
+  - EXTRA_PIP_PACKAGES: for adding some python modules to your user environment.
+  - GCSFUSE_BUCKET: for mounting some google cloud storage bucket as a standard
+    file system.
+
+The other file is `secret_config.yml`_, which specifies cluster specific
+encryption tokens.
+
+The jupyterhub proxy secret token is a random hash, which you can generate as follows:
 
 .. code-block:: bash
 
   $ openssl rand -hex 32
 
-Pangeo.pydata.org uses `GitHub OAuth Callback
-<https://help.github.com/enterprise/2.13/admin/guides/user-management/using-github-oauth/>`_,
-(or `GitHub OAuth for developer <https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/>`_)
+Pangeo.pydata.org uses `GitHub OAuth Callback`_, (or `GitHub OAuth for developers`_)
 to authenticate users. The ``clientSecret`` token needs to be obtained via
-github. 
+github.
 
 This authentication method needs an IP or domain name to work, the IP you've
 reserved above and put in jupyter_config.yaml if you don't have a domain name
```

Comment:
Something is wrong here, this should be somethign like "take the IP you've reserved above". Plus I think there is no pangeo.pydata.ord domain name in the file anymore, is there? I know, that's my writting here...